azext_iot/tests/digitaltwins/test_dt_model_lifecycle_int.py

```diff
@@ -31,6 +32,11 @@ def test_dt_models(self):
         inline_model = "./models/Floor.json"
         room_dtmi = "dtmi:com:example:Room;1"
         floor_dtmi = "dtmi:com:example:Floor;1"
+        ontology_directories = [
+            "./references/opendigitaltwins-building/Ontology",
+            "./references/opendigitaltwins-smartcities/Ontology",
+            "./references/opendigitaltwins-energygrid/Ontology"
```

Comment:
Since the energygrid project is not a configured submodule (currently) it won't be that useful without additional steps, for local or pipeline testing.

Reply:
I added energygrid as a submodule in this PR: https://github.com/Azure/azure-iot-cli-extension/pull/535/files

Reply:
Ah I missed this PR. Does this submodule configuration work? It is missing reference file pointing to the target commit.

Reply:
I just tested it, it doesn't work as is.

Reply:
Add reference file: https://github.com/Azure/azure-iot-cli-extension/pull/537